community.md

```diff
@@ -133,4 +135,5 @@ Below is a list of projects and resources from the Blazor community (links in ea
 ## Other
 
 * [Bionic](https://bionicframework.github.io/Documentation) &ndash; An Ionic CLI clone for Blazor projects.
-* [Trail](https://github.com/panesofglass/trail) &ndash; Blazor rendering with F#.
+* [Bolero](https://github.com/fsbolero/Bolero) &ndash; Blazor for F# with hot reloaded templates, type-safe endpoints and routing, remoting, and much more.
+* [Trail](https://github.com/panesofglass/trail) &ndash; Blazor rendering with F# (now superseded by Bolero).
```

Comment:
Since Train has been superseded by Bolero, let's not add it here. We'd like to focus on actively maintained projects.

Reply:
Done, thanks!